tests/bin/expected/petstore.ts

```diff
@@ -5,458 +5,481 @@
 
 export interface paths {
   '/pet': {
-    put: {
-      requestBody: {
-        'application/json': components['schemas']['Pet']
-        'application/xml': components['schemas']['Pet']
-      }
-      responses: {
-        /**
-         * Invalid ID supplied
-         */
-        '400': unknown
-        /**
-         * Pet not found
-         */
-        '404': unknown
+    put: operations['updatePet']
+    post: operations['addPet']
+  }
+  '/pet/findByStatus': {
+    get: operations['findPetsByStatus']
+  }
+  '/pet/findByTags': {
+    get: operations['findPetsByTags']
+  }
+  '/pet/{petId}': {
+    get: operations['getPetById']
+    post: operations['updatePetWithForm']
+    delete: operations['deletePet']
+  }
+  '/pet/{petId}/uploadImage': {
+    post: operations['uploadFile']
+  }
+  '/store/inventory': {
+    get: operations['getInventory']
+  }
+  '/store/order': {
+    post: operations['placeOrder']
+  }
+  '/store/order/{orderId}': {
+    get: operations['getOrderById']
+    delete: operations['deleteOrder']
+  }
+  '/user': {
+    post: operations['createUser']
+  }
+  '/user/createWithArray': {
+    post: operations['createUsersWithArrayInput']
+  }
+  '/user/createWithList': {
+    post: operations['createUsersWithListInput']
+  }
+  '/user/login': {
+    get: operations['loginUser']
+  }
+  '/user/logout': {
+    get: operations['logoutUser']
+  }
+  '/user/{username}': {
+    get: operations['getUserByName']
+    put: operations['updateUser']
+    delete: operations['deleteUser']
+  }
+}
+
+export interface operations {
+  updatePet: {
+    requestBody: {
+      'application/json': components['schemas']['Pet']
+      'application/xml': components['schemas']['Pet']
+    }
+    responses: {
+      /**
+       * Invalid ID supplied
+       */
+      '400': unknown
+      /**
+       * Pet not found
+       */
+      '404': unknown
+      /**
+       * Validation exception
+       */
+      '405': unknown
+    }
+  }
+  addPet: {
+    requestBody: {
+      'application/json': components['schemas']['Pet']
+      'application/xml': components['schemas']['Pet']
+    }
+    responses: {
+      /**
+       * Invalid input
+       */
+      '405': unknown
+    }
+  }
+  /**
+   * Multiple status values can be provided with comma separated strings
+   */
+  findPetsByStatus: {
```

Comment:
I’m curious how moving this JSDoc affects VS Code, if at all. Not really a concern; just curious (I actually received a lot of comments about this feature! It’s surprisingly popular)

Reply:
good question, I don't know but I'll check it out and report back

Reply:
in VS code the description now only shows up for `operations["findPetsByStatus"]`, but not for `paths["/pet/findByStatus"]["get"]`, which is a bummer ... I feel like that's something that should be fixed in the VS Code integration though ... I wouldn't add the same description to both places. We don't do that when we reference other components either. Let me know what you think

Reply:
Ah gotcha. Yeah I guess you could maybe hoist the comments, but that is one of the most annoying parts of the code. I wouldn‘t worry about it until someone raises it because it‘s not a great payoff